docs/integrations/scalacheck.md

```diff
@@ -117,6 +117,33 @@ class IntegerSuite extends ScalaCheckSuite {
 }
 ```
 
+## Reproducing a property failure
+
+In some cases, you may find that ScalaCheck properties fail
+non-deterministically. This can happen due to the randomness of the input values
+for each test run.
+
+When this happens, you can deterministally reproduce a property failure by using
+its seed. MUnit prints the property seed whenever a failure occurs in the form
+of:
+
+```
+Failing seed: CTH6hXj8ViScMmsO78-k4_RytXHPK_wSJYNH2h4dCpB=
+```
+
+To reproduce the exact same failure, you can override the test parameters and
+provide the offending seed as the initial seed:
+
+```scala
+  override def scalaCheckTestParameters =
```

Comment:
What do you think about making it possible to override the default seed with a string? The format of the output could also be 

override def scalaCheckSeed = “Ceirgejskfh”

So that users can copy paste it

Reply:
implemented, take a look!